code-challenges.test.js

```diff
@@ -37,12 +46,31 @@ const num3 = -7
 
 
 // b) Create the function that makes the test pass.
-
+// Pseudo Code:
+// Create a function called divBy3
+// Make a parameters called num
+// Make a conditional statement
+// Have the conditional be a modulo
+//return each statement with string interpulation
+const divBy3 = (num) => {
+  if (num % 3 === 0){
+    return `${num} is divisible by three`
+  } else {
+    return `${num} is not divisible by three`
+  }
+}
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("randomNouns", () => {
+  it("returns nouns with the first letter capitalized", () => {
+    expect(capitalWords(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capitalWords(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
```

Comment:
Test looks great!  I would recommend calling your test in the describe block the same thing as your function and not your variable name. Also, it is best practice to keep your variables within the same scope that they are used.  In this case, your test variables are used inside your expect statement which is inside your it method, so the variables would be best placed inside your it method.